Formula/zsh-navigation-tools.rb

```diff
@@ -0,0 +1,34 @@
+class ZshNavigationTools < Formula
+  desc "Zsh curses-based tools, e.g. multi-word history searcher"
+  homepage "https://github.com/psprint/zsh-navigation-tools"
+  url "https://github.com/psprint/zsh-navigation-tools/archive/v2.0.5.tar.gz"
+  sha256 "f599a36693b9bacaccdb5ce9939f5be3695b59ef5b84485f618164bb2c77fe95"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  def caveats
+    <<-EOS.undent
+    To run zsh-navigation-tools, add the following at the end of your .zshrc:
+
+      fpath+=( #{HOMEBREW_PREFIX}/share/zsh-navigation-tools )
+      source #{HOMEBREW_PREFIX}/share/zsh-navigation-tools/zsh-navigation-tools.plugin.zsh
+
+    You will also need to force reload of your .zshrc:
+
+      source ~/.zshrc
+
+    You can see all the tools provided by entering "n-" and hitting <Tab> to complete.
+    N-history, the incremental multi-word history searcher, is bound to Ctrl-R.
+    Configuration files are in ~/.config/znt
+    EOS
+  end
```

Comment:
I removed description of software. What's left is instruction to run it, like in zsh-syntax-highlighting package. Can it be that way?